Runtime/UrdfComponents/UrdfRobot.cs

```diff
@@ -49,6 +50,16 @@ public void ChangeToCorrectedSpace(bool rosSpace)
             this.transform.Rotate(0, 180, 0);
         }
 
+        public bool CheckOrientation()
```

Comment:
This one-line function seems only used in one place. Don't have to be a separated function.

Reply:
I want to keep current orientation private so it is not changed from outside except by using this function same for reading it.

Reply:
Okay, if you see the possibility that this function will be reused somewhere else, it is completely fine to make it a separate function.